src/utils/encoding/mod.rs

```diff
@@ -185,3 +189,70 @@ mod tests {
         Ok(())
     }
 }
+
+// Unsigned VarInt (Uvi) Bytes
```

Comment:
I find the lack of tests for a byte juggling features disturbing.

Reply:
Added tests (and fixed a bug)